website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -20,9 +25,10 @@ All dbt Core versions released prior to 1.0 and their version-specific documenta
 
 ## EOL version support 
 
-All dbt Core versions with an end-of-life (EOL) support level will no longer receive bug fixes. To continue receiving bug fixes, dbt Labs recommends upgrading to a newer version. 
+All dbt Core versions with an end-of-life (EOL) support level will no longer receive any fixes in new patch releases. We recommend upgrading to a newer version in [dbt Cloud](/docs/dbt-versions/upgrade-core-in-cloud) or [dbt Core](/docs/core/installation#upgrading-dbt-core) to continue receiving support. 
```

Comment:
I think it's important to call out that it's bug fixes. We will provide security fixes and patches where required. We also occasionally backport non-bug fixes that address backwards compatibility. 
```suggestion
All dbt Core versions with an end-of-life (EOL) support level will no longer receive bug fixes in new patch releases. We recommend upgrading to a newer version in [dbt Cloud](/docs/dbt-versions/upgrade-core-in-cloud) or [dbt Core](/docs/core/installation#upgrading-dbt-core) to continue receiving support. 
```

Reply:
good question @matthewshaver, we originally had bug fixes but @jtcohen6 recommended stronger language [here](https://github.com/dbt-labs/docs.getdbt.com/pull/3532#discussion_r1234213710): "We can make this stronger: They'll no longer receive any fixes in new patch releases."

I do think including bug fixes is helpful. @jtcohen6 @matthewshaver, what about:

>All dbt Core versions with an end-of-life (EOL) support level will no longer receive any fixes, including bug fixes, in new patch releases.

Reply:
That statement is accurate. Here's how I think about it:

> All dbt Core minor versions that have reached end-of-life (EOL) will have no new patch releases. This means they will no longer receive any fixes, including for known bugs that have been identified. Fixes for those bugs will instead be made in newer minor versions that are still under active support.

A lot of passive voice, and a bit redundant — but I think it's important that we drive this point home.

Reply:
sounds great thank you @jtcohen6 and i dont think it's redundant at all! it's very important to very clear about this, i always remember clarifying things that might seem 'obvious' to seasoned people but not so obvious to others so i always like to side with being extra clear.